frontend/packages/ceph-storage-plugin/src/constants/queries.ts

```diff
@@ -12,3 +12,14 @@ export const DATA_RESILIENCY_QUERIES = {
   [StorageDashboardQuery.CEPH_PG_CLEAN_AND_ACTIVE_QUERY]: 'ceph_pg_clean and ceph_pg_active',
   [StorageDashboardQuery.CEPH_PG_TOTAL_QUERY]: 'ceph_pg_total',
 };
+
+export const enum CapacityQueryLabels {
+  CAPACITY_TOTAL = 'capacity_total',
```

Comment:
can you move these values to `StorageDashboardQuery` and follow the same pattern

Reply:
Oh! ok... Sure...

Reply:
Done...